soh/soh/Enhancements/randomizer/draw.cpp

```diff
@@ -10,9 +10,14 @@
 #include "objects/object_gi_hearts/object_gi_hearts.h"
 #include "objects/gameplay_field_keep/gameplay_field_keep.h"
 
+extern "C" u8 Randomizer_GetSettingValue(RandomizerSettingKey randoSettingKey);
+
 extern "C" void Randomizer_DrawSmallKey(PlayState* play, GetItemEntry* getItemEntry) {
     s32 pad;
-    s8 isColoredKeysEnabled = CVarGetInteger("gRandoMatchKeyColors", 1);
+    s8 keysanityEnabled = getItemEntry->getItemId == RG_GERUDO_FORTRESS_SMALL_KEY ?
+        Randomizer_GetSettingValue(RSK_GERUDO_KEYS) != RO_GERUDO_KEYS_VANILLA :
+        Randomizer_GetSettingValue(RSK_KEYSANITY) >= RO_DUNGEON_ITEM_LOC_ANY_DUNGEON;
```

Comment:
when doing the enum stuff for RSKs i tried to move away from using `<` and `>` and instead just use `==` or `!=` (example pr [here](https://github.com/HarbourMasters/Shipwright/pull/2096/files))

it ends up being a bit more verbose, but i feel that tradeoff is worth it to be explicit (both when reading the code, and in case we ever decide to rearrange options)

Reply:
Fair, I'll probably just make a macro if possible as I don't like sacrificing the readability

Reply:
https://github.com/HarbourMasters/Shipwright/compare/7653835c1d5759c98bf20a3c8471225f3cb2f92e..325dd80c96ad5b4ece2bfac2bae3818a6551ad75